utilities/help_utils.py

```diff
@@ -19,6 +20,12 @@ class ModuleHelp:
     The **main segment** contains an obligatory short module description (`descr`) and an optional, longer module description (`longdescr`).
     The main segment is followed by one or more **command segments**, each describing one specific command or a set of related commands:
     what they do and how to use them. A detailed specification of command segments can be found in the docstring of the `CommandHelp` class.
+
+    ### How to add `help` to `Module`
+
+    ```py
+    self.help = ModuleHelp.from_docstring(self.class_name, __doc__)
```

Comment:
and this is not possible to do in `Module` for all modules to inherit?

Reply:
I did put it in `Module` but then by default the children modules initialize with empty help because they use the non-existent docstring from `Module` :shrug:

Reply:
following would work (not sure how to apply in our class structure, but works in https://colab.research.google.com/drive/1-VMAmlEoojGIDOF_zWGdE-xmL7ycXo-j?usp=sharing .. maybe something simpler might work too, maybe not)
```py
class Module():
  @classmethod
  def help(cls):
    return cls.__doc__

class A(Module):
  "gg"
  pass

a = A()
print(a.help())
```

Reply:
Hmm, yeah... I think you pointed me to an idea